mindsdb/integrations/handlers/postgres_handler/tests/docker/certs/server.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Is this OK? 😮

Reply:
yes, it is a self signed certificate which I upload to the docker-container to have ssl connection with test postgres database container

Reply:
Aaah got it 👌